### PR TITLE
Add Doc-How to Create a DevOps Kubeconfig on AWS

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

### CODE REVIEW

File: content/en/docs/faq/devops/create-devops-kubeconfig-on-aws.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+Weight: 16820
+---
+
+This tutorial demonstrates how to create a DevOps kubeconfig on AWS.
+
+## Prerequisites
+
+- You have an AWS cluster with KubeSphere installed. For more information about how to install KubeSphere on AWS, refer to [Deploy KubeSphere on AWS EKS](../../../installing-on-kubernetes/hosted-kubernetes/install-kubesphere-on-eks/).
+- You have enabled [the KubeSphere DevOps system](../../../pluggable-components/devops/).
+- You have created a project. This tutorial uses the project `kubesphere-sample-dev` as an example.
```
Comment (Sherlock113) [inline content/en/docs/faq/devops/create-devops-kubeconfig-on-aws.md:15]: Create a project here or DevOps project? I notice in line 80 you wrote `Log in to your KubeSphere console and go to your DevOps project.`

Comment (Felixnoo) [reply content/en/docs/faq/devops/create-devops-kubeconfig-on-aws.md:15]: The project here is different from a DevOps project. I think the creation of `devops-deploy.yaml` file and the following `kubectl apply` are to enable the deployment of applications into the project `kubesphere-sample-dev` when creating a kubeconfig credential in a DevOps project. I'll add some words at the beginning.

**Comment by Sherlock113** [comment]
@kubesphere/sig-devops Do you guys have any comment on this PR?

**Pull request merged**